### PR TITLE
Allow to require multiple scopes at once

### PR DESCRIPTION
**Pull request by MichalDolata**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,6 +140,7 @@ This can be changed by following config
   conn1 = Phoenix.ConnTest.build_conn()
   conn2 = conn1 |> Plug.Conn.assign(:scopes, ["scope1", "scope2"])
   conn3 = conn1 |> Plug.Conn.assign(:scopes, ["root_scope"])
+  conn4 = conn1 |> Plug.Conn.assign(:scopes, ["scope1"])
```
Comment (amatalai) [inline README.md]: can we  change order to 
```elixir
conn1 = Phoenix.ConnTest.build_conn()
conn2 = conn1 |> Plug.Conn.assign(:scopes, ["scope1"])
conn3 = conn1 |> Plug.Conn.assign(:scopes, ["scope1", "scope2"])
conn4 = conn1 |> Plug.Conn.assign(:scopes, ["root_scope"])
```
so we have more permissions with each conn ?

File: lib/conductor/plugs/authorize.ex [hunk 2]
```diff
@@ -8,29 +8,52 @@ defmodule Conductor.Plugs.Authorize do
...
     action can be performed. Otherwise it will respond according to
     choosen failure strategy (default: raise `Conductor.Error`)
+
+    If you would like to require multiple scopes at once, inclose them in tuple.
+    For example:
+    ```
+    @authorize scope: {"scope1", "scope2"}
+    def action(conn, _params), do: ...
+    ```
+    In order to perform action both "scope1" and "scop2" must be assigned inside conn.
```
Comment (amatalai) [inline lib/conductor/plugs/authorize.ex]: typo: `scop2`

File: README.md (hunk 1 above)
Comment (MichalDolata) [reply README.md]: Sure

Comment (MichalDolata) [reply README.md]: I also changed order of actions to make table look like triangular matrix.

**Pull request merged**